gremlin-client/src/connection.rs

```diff
@@ -108,6 +114,11 @@ impl ConnectionOptionsBuilder {
         self.0.tls_options = Some(options);
         self
     }
+
+    pub fn version(mut self, version: Version) -> Self {
```

Comment:
I would probably rename this to `serializer`

Reply:
I renamed it as suggested in the latest push to the PR branch.